BUILD_ME.md

```diff
@@ -13,95 +13,305 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-```
-sudo apt-get install cmake
-sudo apt-get install bison
-sudo apt-get install flex
-sudo apt-get install binutils-dev
-sudo apt-get install libiberty-dev
-sudo apt-get install slapd
-sudo apt-get install openldap-dev
-sudo apt-get install libicu-dev
-sudo apt-get install libxslt-dev
-sudo apt-get install zlib1g-dev
-sudo apt-get install libarchive-dev
-sudo apt-get install libboost-all-dev
-sudo apt-get install libssl-dev
-sudo apt-get install libapr1-dev
-sudo apt-get install libaprutil1-dev
-sudo apt-get install clang
-sudo apt-get install libtool
-sudo apt-get install autotools-dev
-sudo apt-get install automake
-```
+* Visit [https://github.com/hpcc-systems/HPCC-Platform/wiki](https://github.com/hpcc-systems/HPCC-Platform/wiki) for upto date help and guidance.
 
-* Check out sources (for example, to directory ~/hpcc)
-* Fetch all sub-modules with:
+## Prerequisites
+
+### Ubuntu
+#### Ubuntu 19.10/19.04/18.04
+
+    sudo apt-get install cmake bison flex build-essential binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev zlib1g-dev libboost-regex-dev \
+    libarchive-dev python-dev libv8-dev default-jdk libapr1-dev libaprutil1-dev libiberty-dev libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev nodejs \
+    libevent-dev libatlas-base-dev libblas-dev python3-dev default-libmysqlclient-dev libsqlite3-dev r-base-dev r-cran-rcpp r-cran-rinside r-cran-inline \
+    libmemcached-dev libcurl4-openssl-dev pkg-config libtool autotools-dev automake libssl-dev
+
+#### Ubuntu 16.04
+
+    sudo apt-get install cmake bison flex build-essential binutils-dev libldap2-dev libcppunit-dev libicu-dev libxslt1-dev zlib1g-dev libboost-regex-dev \
+    libssl-dev libarchive-dev python-dev libv8-dev default-jdk libapr1-dev libaprutil1-dev libiberty-dev libhiredis-dev libtbb-dev libxalan-c-dev libnuma-dev \
+    libevent-dev libatlas-base-dev libblas-dev libatlas-dev python3-dev libcurl4-openssl-dev libtool autotools-dev automake
+
+##### Additional information for building versions prior to 7.0 on Ubuntu 18.04 #####
+
+Get openssl-1.0.2o.tar.gz from https://www.openssl.org/source/
+
+unpack, build, and install:
+
+    ./config -fPIC shared
+    make
+    make install
+
+This installs to /usr/local/ssl by default. 
+
+Build the platform with the following additional CMake options:
+
+    -DOPENSSL_LIBRARIES=/usr/local/ssl/lib/libssl.so -DOPENSSL_SSL_LIBRARY=/usr/local/ssl/lib/libssl.so
+
+### CentOS
+
+Regardless of which version of Centos you will be building on, it is suggested that you enable the EPEL repository 
+
+```sudo yum install -y epel-release```
+
+#### CentOS 6.4:
+
+    sudo yum install gcc-c++ gcc make bison flex binutils-devel openldap-devel libicu-devel libxslt-devel libarchive-devel boost-devel openssl-devel \
+    apr-devel apr-util-devel hiredis-devel numactl-devel libmysqlclient-dev libevent-devel tbb-devel atlas-devel python34 R-core-devel R-Rcpp-devel \
+    R-inline R-RInside-devel nodejs libcurl-devel
+
+#### CentOS 7:
```

Comment:
minor: not new (same as wiki) but Ubuntu (rightly I think) the latest version 1st, so probably this should list CentOS 7 before CentOS 6.4 here too (and in Wiki).